projects/coredns/coredns/Makefile

```diff
@@ -1,21 +1,17 @@
-REPO?=coredns
-CLONE_URL?=https://github.com/coredns/$(REPO).git
-GIT_TAG?=$(shell cat GIT_TAG)
-
-ifeq ("$(CLONE_URL)","")
-	$(error No clone url was provided.)	
-endif
-
-ifeq ("$(REPO)","")
-	$(error No repository name was provided.)	
-endif
-
-ifeq ("$(GIT_TAG)","")
-	$(error No git tag was provided.)	
+BASE_DIRECTORY=$(shell git rev-parse --show-toplevel)
+export RELEASE_BRANCH?=1-18
+DEFAULT_RELEASE=$(shell cat $(BASE_DIRECTORY)/release/$(RELEASE_BRANCH)/RELEASE)
+ifneq ("$(PULL_NUMBER)","")
+	export RELEASE?=$(shell expr $(DEFAULT_RELEASE) \* 10000 + $(PULL_NUMBER))
+else
+	export RELEASE?=$(DEFAULT_RELEASE)
 endif
+ARTIFACT_BUCKET?=$(shell cat $(BASE_DIRECTORY)/release/ARTIFACT_BUCKET)
+GIT_TAG?=$(shell cat GIT_TAG)
 
-RELEASE_BRANCH?="1-18"
-RELEASE?="1"
+REPO=coredns
```

Comment:
We are removing the validations and making them hard values now? Is it because you don't see us having to change those values?

Reply:
I think making everything configurable is unwise. This will always just build coredns, nothing else

Reply:
Makes sense